x-pack/filebeat/module/aws/cloudtrail/ingest/pipeline.yml

```diff
@@ -55,6 +55,26 @@ processors:
       field: "json.userIdentity.invokedBy"
       target_field: "aws.cloudtrail.user_identity.invoked_by"
       ignore_failure: true
+  - rename:
+      field: "json.userIdentity.sessionIssuer.type"
+      target_field: "aws.cloudtrail.user_identity.session_issuer.type"
+      ignore_failure: true
+  - rename:
+      field: "json.userIdentity.sessionIssuer.userName"
+      target_field: "user.name"
```

Comment:
Is there a chance this overwrites `user.name` already set from `userIdentity.userName` earlier in the pipeline? Is this what we want?

Reply:
I was worried about that too, I'll add a comment.  sessionIssuer.userName should only be set when there is an assumed role.